src/app/pages/feedback/feedback.page.ts

```diff
@@ -31,21 +32,25 @@ export class FeedbackPage implements OnInit {
     private router: Router,
     private modalController: ModalController,
     private navParams: NavParams,
-    private localStorage: LocalStorageService) {
+    private localStorage: LocalStorageService,
+    private profileService: ProfileService) {
     this.sessionData = this.navParams?.data?.data;
   }
 
   ngOnInit() {
     this.isMentorChecking();
   }
   async isMentorChecking() {
-    let userDetails = await this.localStorage.getLocalData(localKeys.USER_DETAILS);
-    var response = await this.sessionService.getSessionDetailsAPI(this.sessionData._id);
-    this.mentorName = response.mentorName;
+    var response = await this.sessionService.getSessionDetailsAPI(this.sessionData.id);
+    this.mentorName = response.mentor_name;
     this.sessionTitle = response.title;
-    this.isMentor = userDetails?._id == response?.userId ? true : false;
+    this.isMentor = this.profileService.isMentor;
+    for (let i = 0; i < this.sessionData.form.length; i++) {
```

Comment:
Write it as a separate function so that it can be removed quickly once fixed from backend.

And alsi use forof for looping